### PR TITLE
Trying to do that

### PR DESCRIPTION
**Pull request by martinez-hugo**

### CODE REVIEW

File: lightsoff_api/__init__.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ class Place(db.Model):
     report_count = sa.Column(sa.Integer, nullable=False, default=1)
     latitude = sa.Column(sa.Float, nullable=False)
     longitude = sa.Column(sa.Float, nullable=False)
+    contacted = sa.Column(sa.Integer, default=0)
```
Comment (Fenkiou) [inline lightsoff_api/__init__.py:43]: A boolean would be more explicit ;)
```suggestion
    contacted = sa.Column(sa.Boolean, default=False)
```

Comment (Fenkiou) [reply lightsoff_api/__init__.py:43]: Oh my bad, that's a counter.

Maybe name it `contacted_count/times`, your call

File: tests/routes/test_places.py [hunk 2]
```diff
@@ -36,6 +37,7 @@ def test_post_existing_place(client):
         "phone_number": "Some number",
         "latitude": 0.1,
         "longitude": 0.2,
+        "contacted": 0,
```
Comment (Fenkiou) [inline tests/routes/test_places.py:40]: Add tests for your new endpoint and ensure it works correctly (I mean it increase the counter)

File: lightsoff_api/__init__.py [hunk 3]
```diff
@@ -178,5 +180,36 @@ def create_place_review(path: PlacePath, body: PlaceReviewBody):
...
 
+class PlaceContactResponse(BaseModel):
+    code: int = Field(0, description="Status Code")
+    message: str = Field("ok", description="Exception Information")
+
+
+@app.post(
+    "/places/<string:google_place_id>/contact", responses={"200": PlaceContactResponse}
+)
+def update_place_count_of_contact(path: PlacePath):
```
Comment (Fenkiou) [inline lightsoff_api/__init__.py:191]: `def increase_place_contacted_count`?

Same for the endpoint `"/places/<string:google_place_id>/increase_contacted_count"`

File: lightsoff_api/__init__.py [hunk 4]
```diff
@@ -67,6 +68,7 @@ class PlaceBody(BaseModel):
     phone_number: Optional[str]
     latitude: float
     longitude: float
+    contacted: int
```
Comment (Fenkiou) [inline lightsoff_api/__init__.py:71]: I don't think you need this attribute when creating the place ;)

File: tests/routes/test_places.py [hunk 5]
```diff
@@ -10,6 +10,7 @@ def test_post_new_place(client):
...
```
Comment (Fenkiou) [inline tests/routes/test_places.py:13]: Not needed here, neither below IMO

**Review by Fenkiou — CHANGES_REQUESTED** [review]

File: tests/routes/test_places.py (hunk 2 above)
Comment (martinez-hugo) [reply tests/routes/test_places.py:40]: Ok that's done !

File: lightsoff_api/__init__.py (hunk 4 above)
Comment (martinez-hugo) [reply lightsoff_api/__init__.py:71]: That's done !

File: tests/routes/test_places.py (hunk 5 above)
Comment (martinez-hugo) [reply tests/routes/test_places.py]: That's done !

**Review by gowithefloww — APPROVED** [review]

**Pull request merged**